cmd/lsif-gomod/main.go

```diff
@@ -0,0 +1,79 @@
+package main
```

Comment:
Do we need a line of comment for description of what lsif-gomod is/does?

Reply:
Done.